.github/workflows/build.yml

```diff
@@ -381,18 +321,18 @@ jobs:
         save: false
     - name: Configure CPython
       run: |
-        ./configure CFLAGS="-fdiagnostics-format=json" \
-          --config-cache \
-          --enable-slower-safety \
-          --with-pydebug \
-          --with-openssl="$OPENSSL_DIR" \
-          --with-builtin-hashlib-hashes=blake2 \
-          --with-ssl-default-suites=openssl
+        CMD=(./configure CFLAGS="-fdiagnostics-format=json" --config-cache --enable-slower-safety --with-pydebug --with-openssl="$SSL_DIR")
+        if [ "${{ matrix.ssl }}" = "openssl" ]; then
+          "${CMD[@]}"
+        else
+          "${CMD[@]}" --with-builtin-hashlib-hashes=blake2 --with-ssl-default-suites=openssl
+        fi
     - name: Build CPython
-      run: make -j
+      run: make -j4
```

Comment:
any reason to limit to 4 jobs?

Reply:
Doesn't seem to make a difference on CI.

`make -j` -> 1m 2s

https://github.com/python/cpython/actions/runs/16744892365/job/47401008494#step:13:9

`make -j4` -> 1m 2s

https://github.com/python/cpython/actions/runs/16329339951/job/46127705934?pr=136729#step:13:1

Although on this 8-core macOS, `-j` took 1m8s and made the machine somewhat unresponsive, and `-j8` took 28s.